src/libs/ReactNativeLauncher.js

```diff
@@ -244,6 +255,116 @@ class ReactNativeLauncher extends Launcher {
     }
   }
 
+  /**
+   * This method returns all cookies when asked by the running webview(s).
+   * This method is callable by the content script.
+   *
+   * @param {String} options.cookieDomain : Domain's name where to get cookies from.
+   * @param {String} options.konnectorSlug : Slug of the konnector triggering the function.
+   * @param {String} options.idAccount : Id of the account triggering the function.
```

Comment:
can you document the return please? What is the return if there is no cookie? What is the return when there is multiple cookies?